mmv1/products/databasemigrationservice/product.yaml

```diff
@@ -0,0 +1,13 @@
+--- !ruby/object:Api::Product
```

Comment:
```suggestion
# Copyright 2023 Google Inc.
# Licensed under the Apache License, Version 2.0 (the 'License');
# you may not use this file except in compliance with the License.
# You may obtain a copy of the License at
#
#     http://www.apache.org/licenses/LICENSE-2.0
#
# Unless required by applicable law or agreed to in writing, software
# distributed under the License is distributed on an 'AS IS' BASIS,
# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
# See the License for the specific language governing permissions and
# limitations under the License.

--- !ruby/object:Api::Product
```

Reply:
VCR tests won't run if build is failing, build is failing because of the missing license